cluster/src/main/scala/org/apache/pekko/cluster/protobuf/ClusterMessageSerializer.scala

```diff
@@ -57,7 +57,7 @@ private[pekko] object ClusterMessageSerializer {
   val OldGossipEnvelopeManifest = "org.apache.pekko.cluster.GossipEnvelope"
   val OldClusterRouterPoolManifest = "org.apache.pekko.cluster.routing.ClusterRouterPool"
 
-  // is handled on the deserializing side in 2.6.2 and then on the serializing side in 2.6.3
+  // is handled on the deserializing side in Akka 2.6.2 and then on the serializing side in Akka 2.6.3
```

Comment:
I think you can just remove this comment, it's no longer relevant since Pekko is new project